core/src/main/java/org/apache/calcite/prepare/LixToRelTranslator.java

```diff
@@ -97,7 +97,7 @@ public <T> RelNode translate(Queryable<T> queryable) {
   public RelNode translate(Expression expression) {
     if (expression instanceof MethodCallExpression) {
       final MethodCallExpression call = (MethodCallExpression) expression;
-      BuiltInMethod method = BuiltInMethod.MAP.get(call.method);
+      BuiltInMethod method = BuiltInMethod.FUNCTIONS_MAPS.get(call.method);
```

Comment:
because this is a static field in BuiltInMethod to store all the functions. it use 'MAP' name before, so I can't continue to use this name as our spark map method name as you suggested. (I set MAP_FUNCTION previously).  

one disadvantage I can think of here is that it may cause compatibility issues? If there is an upper-layer application that reflects the use of this field. 

or should i reset spark MAP function to 'MAP_FUNCTION' name? what do you think?